.github/workflows/pr-test-lint.yml

```diff
@@ -78,6 +78,9 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
 
+      - name: 'Set up gcloud CLI'
```

Comment:
This is needed to prevent the error:
```
error getting credentials - err: exec: "docker-credential-gcloud": executable file not found in $PATH, out: ``
```